### DISCUSSION TITLE
挂载路径中的某个目录设置别名的功能

### DISCUSSION THREAD

**Discussion by KHwang9883** [AlistGo/alist#6727] [Ideas]
不知道能不能做这样一个功能：挂载路径中的某个目录可以设置一个别名

如：我的 OneDrive 存储挂载在 `/onedrive`，里面有一个叫 `测试` 的文件夹，通过 AList 访问该目录的地址是 `alist网址/onedrive/测试/`。在不对该文件夹重命名的前提下，设置一个 `test` 的别名，这样可以通过 `alist网址/onedrive/test/` 访问到。原来的 `测试` 路径也可以用元信息隐藏，就实现了在 AList 中对某路径更名而不影响存储中目录名称。

**Comment by Simonzhang8** [comment]
你说的这个用alias别名就可以实现啊。配置信息里【挂载路径】填【/onedrive/test】，【路径】填【/onedrive/测试】，就可以达到你说的效果啊，不需要做新的功能。

**Reply by KHwang9883** [reply]
了解了

**Reply by KHwang9883** [reply]
这样做有个问题，别名的目录是显示在最下面也就是文件下面的
![图片](https://github.com/alist-org/alist/assets/44646371/69974066-3b5a-4017-ae75-b0be60baba25)

而没有遵照 `提取文件夹` 的设置
![图片](https://github.com/alist-org/alist/assets/44646371/b68a8bf1-881e-40f1-acde-4f2b19edbac1)
这样设置是没用的

也不是 `序号` 的原因，`序号` 设置为 0 依旧这样
